README.md

```diff
@@ -17,35 +17,40 @@ pgai brings embedding and generation AI models closer to the database. With pgai
 
 * Create [embeddings](#embed) for your data.
 * Retrieve LLM [chat completions](#chat-complete) from models like OpenAI GPT4o and Llama 3.
+* Generate responses for models such as Ollama.
 * Reason over your data and facilitate use cases like [classification, summarization, and data enrichment](docs/advanced.md) on your existing relational data in PostgreSQL.
 
 Here's how to get started with pgai:
 
 * **Everyone**: Use pgai in your PostgreSQL database.
-  * [Installation](#installation)
-  * [Add AI functionality to your database](#usage).
-  * [Advanced AI examples using data](./docs/advanced.md)  
-  * [Address provider-specific setup steps](#provider-specific-setup)
+  1. [Set up pgai](#set-up-pgai).
+  1. Use pgai to integrate AI from your provider:
+    * [Ollama](./docs/ollama.md) - configure pgai with Ollama, then use the model to embed, chat complete and generate. 
+    * [OpenAI](./docs/openai.md) - configure pgai with OpenAI, then use the model to tokenize, embed, chat complete and moderate. This page also includes advanced examples.
 * **Extension contributor**: Contribute to pgai and improve the project.
-  * [Develop and test changes to the pgai extension](./DEVELOPMENT.md)
+  * [Develop and test changes to the pgai extension](./DEVELOPMENT.md).
   * See the [Issues tab](https://github.com/timescale/pgai/issues) for a list of feature ideas to contribute.
 
 **Learn more about pgai:** To learn more about the pgai extension and why we built it, read this blog post [pgai: Giving PostgreSQL Developers AI Engineering Superpowers](http://www.timescale.com/blog/pgai-giving-postgresql-developers-ai-engineering-superpowers).
 
-## Installation
+## Set up pgai
 
 The fastest ways to run PostgreSQL with the pgai extension are to:
 
-* [Use a pre-built Docker container](#use-a-pre-built-docker-container)
-* [Install from source](#install-from-source)
-* [Use a Timescale Cloud service](#use-a-timescale-cloud-service)
+1. Create your Timescale environment. Either:
+   * [Use a pre-built Docker container](#use-a-pre-built-docker-container).
+   * [Install from source](#install-from-source).
+   * [Use a Timescale Cloud service](#use-a-timescale-cloud-service).
```

Comment:
```suggestion
1. Create your database environment. Either:
   * [Use a pre-built Docker container](#use-a-pre-built-docker-container).
   * [Install from source](#install-from-source).
   * [Use a Timescale Cloud service](#use-a-timescale-cloud-service).
```
Avoid "Timescale". For instance, "install from source" does not involve Timescale.